types/simulation/account_test.go

```diff
@@ -21,7 +21,7 @@ func TestRandomAccounts(t *testing.T) {
 	}{
 		{"0-accounts", 0, 0},
 		{"0-accounts", 1, 1},
-		{"0-accounts", 1_000, 1_000},
+		{"0-accounts", 100, 100},
```

Comment:
reduced the number of keys generated from 1000 to 100 because this causes the tests to run for longer than expected. I dont see a difference in 100 and 1000 here other than one take more time, right?

Reply:
Sounds sensible. Maybe 9/10 reduction is a lot, how about trying to halve it first?

Reply:
@jgimeno is there a reason behind picking 1_000, i see you are the most recent person to work on this?

Reply:
@marbar3778 in this file I just removed the dependency for the old x/simulation, so I don't have any opinion on this, Alessio is more familiar with it.

Reply:
@marbar3778 your call Marko, we can try it with 100 - not a blocker anyway IMHO

Reply:
ah found the culprit to the long test, it was a randstrings function testing strings of length 1 million now the overall test time has dropped by 300 seconds....